### ISSUE TITLE
Stuck on "Validating World"

### ISSUE THREAD

**Issue by GoatyMcGoatsAlot** [TEdit/Terraria-Map-Editor#1227]
## What were you doing when the issue occured?
Saving a world, after the menu "The world was externally modified since your last save" even though it wasnt? I click overwrite and

It gets stuck on "validating world" and it never creates or saves the new world file.

It only works properly when I use a blank "New" world from the Tedit menu. But I'd like to alter my existing world. 
## Can you reproduce the issue?
Yes
## What version of Terraria are you using?
1.4.0.3
## Post relevant contents of log or attach log file _**(very important)**_:

World File:
[The_Awkward_Bush.zip](https://github.com/TEdit/Terraria-Map-Editor/files/4670722/The_Awkward_Bush.zip)

Log:
05-22-2020 16:08:31: Starting TEdit 4.0.0.0 
05-22-2020 16:08:31: OS: Microsoft Windows NT 6.2.9200.0 
05-22-2020 16:08:31: Terraria v1.4.0.3 
05-22-2020 16:08:31: Terraria Data Path: D:\SteamLibrary\steamapps\common\Terraria\Content 
05-22-2020 16:08:31: Creating Undo cache: C:\Users\User\AppData\Roaming\TEdit\undo_244674341

**Comment by TheRogue12** [comment]
I have the same issue at Issue #1228

**Comment by TWIKYZ** [comment]
I have the same issue at Issue #1228

**Comment by JDSammy** [comment]
Same comment I wrote over at #1228, but there's been issues with having a few specific entities on your world when trying to edit worlds: if you have any Pylons, Hat Racks, Mannequins (both types), Weapon Racks, or Item Frames, you get the error message that you see there. If it doesn't find any of these entities, I've found that it will usually work. So, try and get rid of each of these entities, then try editing the world. If it doesn't give you an error, it should allow you to be able to edit it.
